src/pavan/LoopWhileExample.java

```diff
@@ -0,0 +1,43 @@
+package pavan;
+
+public class LoopWhileExample {
+	void ModOftwo(int startlimit,int endlimit){
+		int index= startlimit;		
```

Comment:
This extra variable not require, you can handle this without using that variable,
Kindly refer below code for the same.

https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/208/files